assets/scripts/tracking-onboarding.js

```diff
@@ -1,13 +1,9 @@
-(function(w, d) {
-  try {
-    const data = d.currentScript.dataset;
-    const tracker = w.Piwik.getTracker(data.matomoUrl, data.matomoSiteId);
-    tracker.enableHeartBeatTimer();
-    tracker.setUserId(w.location.hostname);
-    tracker.setCustomDimension(data.matomoAppId, 'Onboarding');
-    tracker.setCustomUrl('/password');
-    tracker.trackPageView();
-  } catch (err) {
-    console.log('Cannot initialize Matomo:', err);
-  }
-})(window, document);
+;(function(w, d) {
+  const data = d.currentScript.dataset
+  const tracker = w.Piwik.getTracker(data.matomoUrl, data.matomoSiteId)
+  tracker.enableHeartBeatTimer()
+  tracker.setUserId(w.location.hostname)
+  tracker.setCustomDimension(data.matomoAppId, 'Onboarding')
+  tracker.setCustomUrl('/password')
+  tracker.trackPageView()
```

Comment:
I'd prefer to keep the try/catch to avoid showing an error in the console when piwik is not configured